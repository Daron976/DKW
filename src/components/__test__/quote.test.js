import renderer from 'react-test-renderer';
import Quote from '../quote';

describe('Quote component', () => {
  test('header renders as exepcted', async () => {
    const snapQuote = renderer.create(<Quote />)
      .toJSON();
    expect(snapQuote).toMatchSnapshot();
  });
});
