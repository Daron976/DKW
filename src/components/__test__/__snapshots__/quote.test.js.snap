// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component header renders as exepcted 1`] = `
<section
  className="quote-content"
>
  <div
    className="quote-header"
  >
    <h1>
      Quote of the Day
    </h1>
  </div>
  <div
    className="text-container"
  >
    <p
      className="text-content"
    >
      Life is a math equation. In order to gain the most, you have to know how to convert negatives into positives.
    </p>
    <p
      className="text-content"
    >
      <strong>
        <em>
          â€” Anonymous
        </em>
      </strong>
    </p>
  </div>
</section>
`;
